jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/build/BuildConfiguration.java

```diff
@@ -56,67 +56,169 @@ public class BuildConfiguration implements Serializable {
   public static final String DEFAULT_FILTER = "${*}";
   public static final String DEFAULT_CLEANUP = "try";
 
+  // Generic fields applicable to all build strategies
+  /**
+   * The base image which should be used for this image.
+   *
+   * <p> If not given this default to <code>busybox:latest</code> and is suitable for a pure data image.
+   * <p>
+   * This field is applicable only for all build strategies.
+   */
+  private String from;
+  /**
+   * Extended definition for a base image. This field holds a map of defined in key:value format.
+   * <p> The known keys are:
+   * <ul>
+   *   <li><b>name</b>: Name of the base image</li>
+   * </ul>
+   * <p> A provided {@link BuildConfiguration#from} takes precedence over the name given here.
+   * This tag is useful for extensions of this plugin.
+   * <p>
+   * This field is applicable only for all build strategies.
```

Comment:
nit: for this and every other field, `This field is applicable only...`, the `only` here is not OK